src/test/java/com/google/firebase/auth/UserImportHashTest.java

```diff
@@ -145,6 +146,91 @@ public void testBasicHash() {
     }
   }
 
+  private void assertBuilderThrowsIllegalArgumentException(Md5.Builder builder) {
+    try {
+      builder.build();
+      fail("Expected IllegalArgumentException to be thrown but no expecption occurred.");
+    } catch (IllegalArgumentException expected) {
+    }
+  }
+
+  private void assertBuilderThrowsIllegalArgumentException(Sha1.Builder builder) {
```

Comment:
If you decide to keep these helpers, move them further down in the file, after all the public methods.

Reply:
(no longer applicable.)